translations/messages.fr.yaml

```diff
@@ -3,5 +3,6 @@ You have SUNK a ship! Well done!: Vous avez coulé un bateau ! Bien joué !
 You hit a ship!: Vous avez touché un bateau !
 You hit the water...: Vous avez tiré dans l'eau...
 Abandoned Game: Partie abandonnée
-YOUR TURN: VOTRE TOUR
-WAITING FOR YOUR OPPONENT: EN ATTENTE DE L'AUTRE JOUEUR
\ No newline at end of file
+Your Turn: Votre tour
+Waiting for your opponent: En attente de l'autre joueur
+Win Game: Partie gagnée !
```

Comment:
Great! If you want to uppercase it in twig you can use `upper` like this `{{ "Win Game"|trans|upper }}`

Reply:
Thanks for the pro tip :+1: